src/main/java/lk/gov/health/phsp/bean/ClientController.java

```diff
@@ -461,7 +455,9 @@ public String toRegisterdClientsWithDatesForSystemAdmin() {
     }
 
     public void fillClients() {
-        String j = "select new lk.gov.health.phsp.pojcs.ClientBasicData("
+        Map param_map = new HashMap();
```

Comment:
Its a local variable. Used the same way in thousands of places. Better to leave this as it is.